platform/security/sts/security-sts-propertyclaimshandler/src/test/java/org/codice/ddf/security/sts/claims/property/TestAttributeFileClaimsHandler.java

```diff
@@ -158,6 +160,42 @@ public void testRetrieveClaimsValuesNullPrincipal() {
     ProcessedClaimCollection processedClaims =
         attributeFileClaimsHandler.retrieveClaimValues(claimCollection, claimsParameters);
 
-    Assert.assertThat(processedClaims.size(), CoreMatchers.is(equalTo(0)));
+    assertThat(processedClaims, is(empty()));
+  }
+
+  @Test
+  public void testGetSingleValuedSystemHighAttribute() throws IOException {
+    assertThat(attributeFileClaimsHandler.getValues("test"), containsInAnyOrder("testValue"));
+  }
+
+  @Test
+  public void testGetMultiValuedSystemHighAttribute() throws IOException {
+    assertThat(
+        attributeFileClaimsHandler.getValues("test1"),
+        containsInAnyOrder("testing1", "testing2", "testing3"));
+  }
+
+  @Test
+  public void testNoMatchGetSystemHighAttribute() throws IOException {
+    assertThat(attributeFileClaimsHandler.getValues("reg"), is(empty()));
+  }
+
+  @Test(expected = Exception.class)
+  public void testExceptionWhenCantFindSystemHighUserInUsersDotAttributesFile() throws IOException {
+    System.setProperty(SystemBaseUrl.HOST, "someUserThatIsntInTheUsersDotAttributesFile");
+    attributeFileClaimsHandler.init();
+  }
+
+  @Test(expected = Exception.class)
+  public void testExceptionWhenNoUsersDotAttributesFile() throws IOException {
+    attributeFileClaimsHandler.setAttributeFileLocation("noFileHere");
+  }
+
+  @Test(expected = Exception.class)
+  public void testExceptionWhenUnexpectedFormatInUsersDotAttributesFile() throws IOException {
+    userFile = temporaryFolder.newFile("notExpectedFormatUsers.attributes");
+    FileUtils.write(userFile, "not json content in expected format", Charset.defaultCharset());
+
+    attributeFileClaimsHandler.setAttributeFileLocation(userFile.getAbsolutePath());
   }
```

Comment:
❗️ We should also add the following tests:
* `init()` with an invalid file
* Invalid attribute type in JSON map (e.g., integer) to cover L169-175
* `setAttributeFileLocation(null)`
* `getValues(null)`